language/move-compiler/tests/move_check/parser/spec_parsing_inside_fun.exp

```diff
@@ -1,54 +1,9 @@
-error[E04003]: built-in operation not supported
-   ┌─ tests/move_check/parser/spec_parsing_inside_fun.move:32:9
-   │
-32 │         spec {} + 1;
-   │         ^^^^^^^
-   │         │
-   │         Invalid argument to '+'
-   │         Found: '()'. But expected: 'u8', 'u64', 'u128'
-
-error[E04007]: incompatible types
+error[E01002]: unexpected token
    ┌─ tests/move_check/parser/spec_parsing_inside_fun.move:32:17
    │
 32 │         spec {} + 1;
-   │         ------- ^ - Found: integer. It is not compatible with the other type.
-   │         │       │  
-   │         │       Incompatible arguments to '+'
-   │         Found: '()'. It is not compatible with the other type.
-
-error[E04003]: built-in operation not supported
-   ┌─ tests/move_check/parser/spec_parsing_inside_fun.move:32:19
-   │
-32 │         spec {} + 1;
-   │         -------   ^ Invalid argument to '+'
-   │         │          
-   │         Found: '()'. But expected: 'u8', 'u64', 'u128'
-
-error[E04007]: incompatible types
-   ┌─ tests/move_check/parser/spec_parsing_inside_fun.move:33:9
-   │
-33 │         spec {} && spec {};
-   │         ^^^^^^^ -- Expected: 'bool'
-   │         │        
-   │         Invalid argument to '&&'
-   │         Given: '()'
-
-error[E04007]: incompatible types
-   ┌─ tests/move_check/parser/spec_parsing_inside_fun.move:33:20
-   │
-33 │         spec {} && spec {};
-   │                 -- ^^^^^^^
-   │                 │  │
-   │                 │  Invalid argument to '&&'
-   │                 │  Given: '()'
-   │                 Expected: 'bool'
-
-error[E04004]: expected a single non-reference type
-   ┌─ tests/move_check/parser/spec_parsing_inside_fun.move:34:9
-   │
-34 │         &mut spec {};
-   │         ^^^^^^^^^^^^
-   │         │    │
-   │         │    Expected a single non-reference type, but found: '()'
-   │         Invalid borrow
+   │                 ^
+   │                 │
+   │                 Unexpected '+'
+   │                 Expected an expression term
```

Comment:
I am actually a little bit confused that why a `Spec` could be a member of `Expression` https://github.com/move-language/move/blob/164bf33c8b396b9a29305d9c1b9a0d3b0c5b8cef/language/move-compiler/src/parser/ast.rs#L643-L644. Maybe I missed something that is important. Please let me know.

I chose to let `cst` behaved similarly with `ast` which `Exp` includes `Spec` to make the test https://github.com/move-language/move/blob/164bf33c8b396b9a29305d9c1b9a0d3b0c5b8cef/language/move-compiler/tests/move_check/typing/spec_block_fail.move#L1-L7 pass.

I'd like to let this fail earlier in the parsing which `spec` doesn't work like `Exp` . 

Please let me know your ideas on it.